core/tests/upgrade-test/tests/upgrade.test.ts

```diff
@@ -7,40 +7,32 @@ import fs from 'fs';
 import { BytesLike } from '@ethersproject/bytes';
 import { IZkSyncHyperchain } from 'zksync-ethers/build/typechain';
 import { BigNumberish } from 'ethers';
+import { loadConfig, shouldLoadConfigFromFile } from 'utils/build/file-configs';
+import {
+    Contracts,
+    initContracts,
+    runServerInBackground,
+    setAggregatedBlockExecuteDeadline,
+    setAggregatedBlockProveDeadline,
+    setBlockCommitDeadlineMs,
+    setEthSenderSenderAggregatedBlockCommitDeadline
+} from './utils';
+import path from 'path';
 
-const L1_CONTRACTS_FOLDER = `${process.env.ZKSYNC_HOME}/contracts/l1-contracts/artifacts/contracts`;
-const L1_DEFAULT_UPGRADE_ABI = new ethers.Interface(
-    require(`${L1_CONTRACTS_FOLDER}/upgrades/DefaultUpgrade.sol/DefaultUpgrade.json`).abi
-);
-const GOVERNANCE_ABI = new ethers.Interface(
-    require(`${L1_CONTRACTS_FOLDER}/governance/Governance.sol/Governance.json`).abi
-);
-const ADMIN_FACET_ABI = new ethers.Interface(
-    require(`${L1_CONTRACTS_FOLDER}/state-transition/chain-interfaces/IAdmin.sol/IAdmin.json`).abi
-);
-const CHAIN_ADMIN_ABI = new ethers.Interface(
-    require(`${L1_CONTRACTS_FOLDER}/governance/ChainAdmin.sol/ChainAdmin.json`).abi
-);
-const L2_FORCE_DEPLOY_UPGRADER_ABI = new ethers.Interface(
-    require(`${process.env.ZKSYNC_HOME}/contracts/l2-contracts/artifacts-zk/contracts/ForceDeployUpgrader.sol/ForceDeployUpgrader.json`).abi
-);
-const COMPLEX_UPGRADER_ABI = new ethers.Interface(
-    require(`${process.env.ZKSYNC_HOME}/contracts/system-contracts/artifacts-zk/contracts-preprocessed/ComplexUpgrader.sol/ComplexUpgrader.json`).abi
-);
-const COUNTER_BYTECODE =
-    require(`${process.env.ZKSYNC_HOME}/core/tests/ts-integration/artifacts-zk/contracts/counter/counter.sol/Counter.json`).deployedBytecode;
```

Comment:
everywhere with `ZKSYNC_HOME` use `pathToHome = path.join(__dirname, '../../../..`

Reply:
Addressed in https://github.com/matter-labs/zksync-era/commit/0f31c9ff8b170ebd32aa9fe32f99458a46610628